iogt/static/css/modules/_m-article.scss

```diff
@@ -1,46 +1,50 @@
 
-.article-list {
-  list-style: none;
-  padding: 0;
-  margin: 0;
-}
 .article-list__item {
-  padding: 24px;
+  position: relative;
   border-bottom: solid 1px #e4e4e4;
   overflow: auto;
+  a {
+    display: block;
+    color: #666666;
+  }
 }
-.article-list__image {
-  width: 40%;
-  height: auto;
-  margin: 0 24px 9px 0;
-  float: left;
+
+.article__title {
+  color: #000;
+  font-size: $font-medium;
+  line-height: 1.1;
+  padding: 7px 0;
 }
-.article-list__subtitle {
-  margin-bottom: 0;
+
+.article__category--title {
```

Comment:
Is this a modifier, or a new style? Eg: article__category-title? 

Is a category the same thing as a section?